packages/@aws-cdk/aws-cloudtrail/test/cloudtrail.test.ts

```diff
@@ -255,7 +255,6 @@ describe('cloudtrail', () => {
       test('enabled and custom retention', () => {
         const stack = getTestStack();
         new Trail(stack, 'MyAmazingCloudTrail', {
-          sendToCloudWatchLogs: true,
```

Comment:
While we should have at least one test case where this is removed and the other relevant fields are set, making this true, we should also still have a test case where this is set explicitly. Overall, I'd like to see a bit more test coverage for the various cases with this new behavior (including throwing an error as suggested above).